tests/local-inputs-compressed/builds.yaml

```diff
@@ -0,0 +1,23 @@
+inputs:
+  - name: test-local-compressed-asia-sequences
+    metadata: tests/local-inputs-compressed/data/asia_metadata.tsv.xz
+    sequences: tests/local-inputs-compressed/data/asia_sequences.fasta.xz
```

Comment:
These files are excellent! How did you generate them? Do you have a script to automate the process? I'm just thinking about how out of date the current "example data" are and how they are missing metadata fields we may soon add (`aa_substitutions`) and have fields we no longer need (like `*_exposure`). It would be nice if we could quickly regenerate these test files from more recent data as we need to...

Reply:
Yeah - that would be really useful. I just manually split the example metadata up into 5 files & used `augur filter` to generate corresponding sequence files. The benefit of having region-specific files was that I could eyeball the build and see whether genomes from each input made it through. 

It should be easy to make a small script to generate these, but probably best for a subsequent PR.